ISISScientificComputing/autoreduce
# Enginx event mode

### Summary of work
* Ensure that a mechanism exists in the queue processors to enable runs to be skipped
* Ensure that this is updated correctly in the database
* Ensure that skipped runs are displayed correctly in the webapp

### How to test your work
* Checkout this branch on the development environment for the compute node. 
* Submit a run to the development environment that is an Event mode EnginX file (`304652.nxs-304662.nxs`)
* Ensure the run is processed and the result is a run that is in the "Skipped" Status.
* Ensure that this renders correctly in the webapp 

Fixes #301
